cypress/elements/Chart.js

```diff
@@ -0,0 +1,113 @@
+import {
+    VIS_TYPE_COLUMN,
+    VIS_TYPE_GAUGE,
+    VIS_TYPE_PIE,
+    VIS_TYPE_PIVOT_TABLE,
+    VIS_TYPE_SINGLE_VALUE,
+    VIS_TYPE_YEAR_OVER_YEAR_COLUMN,
+    VIS_TYPE_YEAR_OVER_YEAR_LINE,
+} from '@dhis2/analytics'
+
+const visualizationContainerEl = 'visualization-container'
+const visualizationTitleEl = 'visualization-title'
+const visualizationSubtitleEl = 'visualization-subtitle'
+const chartContainerEl = '.highcharts-container'
+const highchartsLegendEl = '.highcharts-legend'
+const highchartsTitleEl = '.highcharts-title'
+//const higchartsLegendItemEl = '.highcharts-legend-item'
+const unsavedVisualizationTitleText = 'Unsaved visualization'
+const AOTitleEl = 'AO-title'
```

Comment:
Sometimes constants are AO_TITLE_EL, other times AOTitleEl. Maybe aim for consitency (I'm guilty of this too).

Reply:
I'm yet to work out a good rule for this pattern, but my thought was that constants that are just picked arbitrarily to run the test (e.g. a data element, a vis type or a name) use SCREAMING_SNAKE (prefixed with TEST_ to make it even more obvious), but anything that's "a regular" const (a return value, expected output, the current item in a for loop etc) use our regular camelCase. 
I find that it's important to separate arbitrary consts from the rest, since the fact that they were indeed just picked arbitrarily makes them available to be changed, made in to an array and be looped through etc.
However I've been quite inconsistent with this convention so Im updating it now.

Reply:
the above applies to `/integration` files. in `/elements` our regular pattern of STRING_CONSTS and regularConsts should be used. However I camelCase and suffix all element names with `El`, not sure this is the best way though, open for suggestions 🙂